app/uk/gov/hmrc/nisp/controllers/redirect/RedirectController.scala

```diff
@@ -16,14 +16,16 @@
 
 package uk.gov.hmrc.nisp.controllers.redirect
 
-import play.api.mvc.{Action, AnyContent}
+import com.google.inject.Inject
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.nisp.utils.Constants
-import uk.gov.hmrc.play.frontend.controller.{FrontendController, UnauthorisedAction}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 
 import scala.concurrent.Future
 
-object RedirectController extends FrontendController  {
-  def redirectToHome(path: String): Action[AnyContent] = UnauthorisedAction.async { implicit request =>
+class RedirectController @Inject()(cc: MessagesControllerComponents) extends FrontendController(cc)  {
+
+  def redirectToHome(path: String): Action[AnyContent] = Action.async { implicit request =>
     val newPath = path match {
       case "" => ""
```

Comment:
At the minute, the code suggests that there are only ever three string variants that will be encountered. It might be worth checking this in the refactor phase as there is no default case.